### PR TITLE
Feedback for new documentation

### PR DESCRIPTION
**Pull request by theskupi**
Feedback for new documentation related to version 5 of Phoenix components.

### CODE REVIEW

File: src/docs/DesignTokens.stories.mdx [hunk 1]
```diff
@@ -39,6 +39,10 @@ Phoenix components ship with a default style, i.e. default set of tokens. These
 
 Below you can see default tokens grouped by type. Because these tables are generated from SCSS file, variable name format is different from the usage in code. In your TypeScript code you should use it with dot notation and without $ sign. E.g. `$ref-color-purple-2` turns into `ref.color.purple.2`
 
+```
+color: ${({ theme }): string => theme.ref.color.purple.2};
+```
+
```
Comment (theskupi) [inline src/docs/DesignTokens.stories.mdx:45]: I added code preview for design token usage. Is this correct?

File: .storybook/preview.js [hunk 2]
```diff
@@ -7,11 +7,10 @@ import theme from '../src/theme'
...
 	options: {
-		/** Sort alphabetically */
-		storySort: (a, b) =>
-			a[1].kind === b[1].kind
-				? 0
-				: a[1].id.localeCompare(b[1].id, undefined, { numeric: true })
+		storySort: {
+			method: 'alphabetical',
+			order: ['Foundation', 'Components', 'Demo']
+		}
```
Comment (theskupi) [inline .storybook/preview.js:13]: From my pov, there is important stuff in Foundation category and I believe that it should be visible on the first look. What the you think about moving it above Components category?

Comment (samuelg0rd0n) [reply .storybook/preview.js:13]: Brilliant idea! Yes 😊 Thanks

File: src/docs/DesignTokens.stories.mdx (hunk 1 above)
Comment (samuelg0rd0n) [reply src/docs/DesignTokens.stories.mdx:45]: Actually, it should be

```
color: ${({ theme }): string => theme.tokens.ref.color.purple.2};
```

`tokens` is missing

Comment (theskupi) [reply src/docs/DesignTokens.stories.mdx:45]: Ok, will do :)

**Pull request merged**